packages/react-native-codegen/src/parsers/error-utils.js

```diff
@@ -0,0 +1,37 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ */
+
+('use strict');
+
+import type {ParserType} from './errors';
+
+const {IncorrectModuleRegistryCallArityParserError} = require('./errors');
+
+function throwIfIncorrectModuleRegistryCallArityParserError(
```

Comment:
What about calling this function `throwIfWrongNumberOfCallExpressionArgs`? I think that it explains better what the method will do: if there is a wrong number of argument in the call expression, it throws.

What do you think?